packages/core/src/components/essentials/legend.ts

```diff
@@ -94,7 +94,45 @@ export class Legend extends Component {
 			addedLegendItemsText.html((d) => d.name);
 		}
 
-		this.breakItemsIntoLines(addedLegendItems);
+		const radiusLabel = this.configs.find(item => 
```

Comment:
configs should be an object, so that we can add more configs in the future to `legend` if wee need to. And the point of adding an `extraLabels` flag to `Legend` would be to remove any `radius` specific logic from `Legend`... You still have everything specific to radius in here